app/robots.ts

```diff
@@ -6,6 +6,7 @@ export default function robots(): MetadataRoute.Robots {
     rules: {
       userAgent: '*',
       allow: '/',
+      disallow: ['/global-components/', '/test/'],
```

Comment:
@sherakama The robot blocking folders

Reply:
Those shouldn't be built should they? It's fine for belt and suspenders here but I don't think these paths should have valid routes.

Reply:
Yeah that's what I'm thinking @sherakama  - maybe we filter those folders out in page.tsx, but maybe leave them in for the editor route so for preview? What would you recommend?